### PR TITLE
Add byebug as debugger

### PR DESCRIPTION
**Pull request by mgrassotti**
Add `byebug` for runtime debugging

ref: #344

### CODE REVIEW

**Comment by 0crat** [comment]
@mgrassotti[/z](https://www.0crat.com/u/mgrassotti) this pull request is too small, just 3 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @yegor256[/z](https://www.0crat.com/u/yegor256) please review this and merge or reject

**Comment by yegor256** [comment]
@mgrassotti why not into `zold.gemspec`?

**Comment by mgrassotti** [comment]
@yegor256 that's fine for me, I pushed a new version

File: zold.gemspec [hunk 1]
```diff
@@ -71,4 +71,5 @@ and suggests a different architecture for digital wallet maintenance.'
   s.add_development_dependency 'rdoc', '4.2.0'
   s.add_development_dependency 'rspec-rails', '3.1.0'
   s.add_development_dependency 'webmock', '3.4.2'
+  s.add_development_dependency 'byebug'
```
Comment (yegor256) [inline zold.gemspec]: @mgrassotti why don't you name the version of it, like it's done for all other dependencies? Any particular reason?

**Comment by yegor256** [comment]
@mgrassotti see above

**Comment by mgrassotti** [comment]
@yegor256 you're right, it's better to specify it

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor mertge

**Comment by mgrassotti** [comment]
@yegor256 I fixed the rubocop offense

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zold-io/zold#345` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope

**Pull request closed without merging**